langserve/api_handler.py

```diff
@@ -1229,6 +1243,18 @@ async def _stream_log() -> AsyncIterator[dict]:
                             "data": self._serializer.dumps(data).decode("utf-8"),
                             "event": "data",
                         }
+
+                    # Send a metadata event as soon as possible
+                    if not has_sent_metadata and self._enable_feedback_endpoint:
```

Comment:
If `_token_feedback_enabled` is `False` but `_enable_feedback_endpoint` is `True`, `task` will be `None`, causing `AssertionError("Feedback token task was not created.")` always raise. Do you mean to check 
`_token_feedback_enabled` here?
Can be reproduced by using the default example in https://github.com/langchain-ai/langserve/blob/109445b25ef047e1cc1624a83eda2deef03a6c83/README.md?plain=1#L399-L406

Reply:
Fixed at https://github.com/langchain-ai/langserve/pull/620